issue.go

```diff
@@ -649,6 +656,32 @@ func (s *IssueService) Create(issue *Issue) (*Issue, *Response, error) {
 	return responseIssue, resp, nil
 }
 
+// UpdateWithOptions updates an issue from a JSON representation,
+// while also specifiying query params. The issue is found by key.
+//
+// JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/issue-editIssue
+func (s *IssueService) UpdateWithOptions(issue *Issue, opts *UpdateOptions) (*Issue, *Response, error) {
```

Comment:
Do you mind naming this `UpdateQueryOptions`?  It matches other parts of the code better (https://github.com/andygrunwald/go-jira/blob/master/issue.go#L528).